stdpopsim/homo_sapiens.py

```diff
@@ -696,3 +696,95 @@ def __init__(self):
             msprime.MassMigration(
                 time=T_nean_split, source=3, destination=0, proportion=1.0)
         ]
+
+
+class SchiffelsZigzag(models.Model):
+    """
+    Model Name:
+        SchiffelsZigzag
+
+    Model Description:
+        A validation model used by Schiffels and Durbin (2014) and Terhorst and Terhorst, Kamm, and Song (2017) with periods
+        of exponential growth and decline in a single population.
+        The original SCRM command for this model was:
+
+        .. code_block::
+
+            scrm 8 1 -N0 14312 -t 1431200 -r 400736 2000000000
+            -eN 0 1 -eG 0.000582262 1318.18 -eG 0.00232905 -329.546 -eG 0.00931619 82.3865
+            -eG 0.0372648 -20.5966 -eG 0.149059 5.14916 -eN 0.596236 0.1 -seed 1 -T -L -p 10 -l 300000.
+
+    Model population indexes:
+        - Single population: 0
+
+    CLI help:
+        python -m stdpopsim homo-sapiens SchiffelsZigzag -h
+
+    Citation:
+       Schiffels, S., & Durbin, R. (2014). Inferring human population size and separation history from multiple genome sequences. Nature Genetics. https://doi.org/10.1038/ng.3015
+    """  # noqa: E501
```

Comment:
I think we should remove the ``noqa`` here, there's no need for long lines in this case.